src/herbpy/herbrobot.py

```diff
@@ -30,6 +34,32 @@ def __init__(self, left_arm_sim, right_arm_sim, right_ft_sim,
 
         Robot.__init__(self, robot_name='herb')
 
+
+        # Controller setup
+        # TODO load controller names from same config as ros_control?
```

Comment:
I am skeptical that this is possible - we have a pretty big set of arbitrary rules about when controllers need to be loaded. If you think this is possible, please file it as an issue so we can track it.